From aadce2662253b88096b3f0abd97b699cc1019579 Mon Sep 17 00:00:00 2001
From: Finley Xiao <finley.xiao@rock-chips.com>
Date: Fri, 2 Nov 2018 10:32:30 +0800
Subject: [PATCH] arm64: dts: rockchip: rk1808: Add dynamic-power-coefficient
 for cpu

The average value of dynamic-power-coefficient is about 74.

Change-Id: I71906da6bd024022b4b64c4dd4fb9f03182fd4e8
Signed-off-by: Finley Xiao <finley.xiao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/rk1808.dtsi | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/boot/dts/rockchip/rk1808.dtsi b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
index 0205873a31d5..4ddcf2b7b2c0 100644
--- a/arch/arm64/boot/dts/rockchip/rk1808.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk1808.dtsi
@@ -47,6 +47,7 @@
 			enable-method = "psci";
 			clocks = <&cru ARMCLK>;
 			operating-points-v2 = <&cpu0_opp_table>;
+			dynamic-power-coefficient = <74>;
 		};
 
 		cpu1: cpu@1 {
@@ -56,6 +57,7 @@
 			enable-method = "psci";
 			clocks = <&cru ARMCLK>;
 			operating-points-v2 = <&cpu0_opp_table>;
+			dynamic-power-coefficient = <74>;
 		};
 	};
 
-- 
2.35.3

